fn_risk_fabric/fn_risk_fabric/components/get_action_plans.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# pragma pylint: disable=unused-argument, no-self-use
+"""Function implementation"""
+
+import logging
+from resilient_circuits import ResilientComponent, function, handler, StatusMessage, FunctionResult, FunctionError
+import risk_fabric
```

Comment:
You could move the 'risk_fabric.py' file into the 'util' directory or a new 'lib' directory, then replace these imports with `from fn_risk_fabric.util.risk_fabric import xxxx`.  Personally I like that because it keeps the components directory cleaner.  But completely optional.